packages/grafana-data/src/types/alerts.ts

```diff
@@ -11,12 +11,11 @@ export enum AlertState {
 }
 
 /**
- * @internal -- might be replaced by next generation Alerting
+ * @internal
  */
 export interface AlertStateInfo {
   id: number;
   dashboardId: number;
   panelId: number;
   state: AlertState;
-  newStateDate: string;
```

Comment:
This is the only thing I worry about, can you explain why this was removed?

Reply:
wasn't used anywhere and we don't have it with new alerting, so I just removed it. Is it needed for something?

Reply:
I just realized I added that 🙈  I think it should be ok to delete